### PR TITLE
Prevent values bleeding between EditableTexts of panel headers

### PR DESCRIPTION
**Pull request by roonyh**
panel title is upadated on enter instead of every key press

### CODE REVIEW

File: modules/web/js/ballerina/components/editable-text.jsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ class EditableText extends React.Component {
 
     renderTextBox() {
         const {x, y, width, height = 25, onChange, onBlur, onKeyDown, children="",
-            className = 'text-input', placeHolder, canUpdate} = this.props;
+            className='text-input', placeholder} = this.props;
```
Comment (hemikak) [inline modules/web/js/ballerina/components/editable-text.jsx]: There should be spacing around the '=' sign.

File: modules/web/js/ballerina/components/panel-decorator.jsx [hunk 2]
```diff
@@ -57,14 +63,26 @@ class PanelDecorator extends React.Component {
...
+    }
+
+    onTitleKeyDown(e) {
+        if(e.keyCode === 13) {
+            const modelType = this.props.model.type.replace('Definition', '');
 
-        // Setter functions take form 'setModelTypeName'. eg: setServiceName
-        this.props.model[`set${modelType}Name`](e.target.value);
+            // Setter functions take form 'setModelTypeName'. eg: setServiceName
+            this.props.model[`set${modelType}Name`](this.state.editingTitle);
```
Comment (hemikak) [inline modules/web/js/ballerina/components/panel-decorator.jsx]: A bit risky to use a template to find and all a function. Can we pass the setter method as a prop?

File: modules/web/js/ballerina/components/utils/tag-component.jsx [hunk 3]
```diff
@@ -68,14 +68,18 @@ class TagController extends React.Component {
      * */
     onKeyDown(e) {
         if (e.keyCode === 13) {
+            let validate = this.props.validateInput;
+            let variableDeclaration = this.state.editValue.replace("=", "").replace(";", "");
```
Comment (hemikak) [inline modules/web/js/ballerina/components/utils/tag-component.jsx]: We need to use single quotes here.

File: modules/web/js/ballerina/components/panel-decorator.jsx (hunk 2 above)
Comment (sumuditha-viraj) [inline modules/web/js/ballerina/components/panel-decorator.jsx]: Agree with Hemika.

**Pull request merged**